Cargo.toml

```diff
@@ -12,9 +12,9 @@ Random number generators and other randomness functionality.
 """
 keywords = ["random", "rng"]
 categories = ["algorithms", "no-std"]
-exclude = ["/utils/*", "/.travis.yml", "/appveyor.yml", ".gitignore"]
 autobenches = true
 edition = "2018"
+include = ["src/**/*", "LICENSE-*", "README.md", "CHANGELOG.md", "COPYRIGHT"]
```

Comment:
Shouldn't we just match `src` here?

Reply:
Sounds good! I have made it 'src/' which has the same effect but makes clearer that it's a directory.
If that's not alright, please let me know and I will make it `src` as originally requested.